packages/admin/cms-admin/src/builds/BuildEntry.tsx

```diff
@@ -145,6 +145,7 @@ export function BuildEntry(): React.ReactElement {
 }
 
 const Content = styled(Box)`
+    padding: 4px 0;
```

Comment:
Shouldn't this be `theme.spacing(1, 0)`?

Reply:
https://github.com/vivid-planet/comet/pull/2541/commits/2ec324d08adf6618f372879ce3d032a6fca136b5